DeepFried2/layers/BackwardsConvolutionCUDNN.py

```diff
@@ -3,50 +3,59 @@
 from theano.sandbox.cuda.basic_ops import gpu_contiguous, gpu_alloc_empty
 from theano.sandbox.cuda import dnn
 
+import numpy as np
+
 
 class BackwardsConvolutionCUDNN(df.Module):
-    def __init__(self, n_input_plane, n_output_plane, k_w, k_h, d_w=1, d_h=1, pad_w=0, pad_h=0, mode='cross', with_bias=True, initW=df.init.xavier(), initB=df.init.const(0), border=None):
+    def __init__(self, n_input_plane, n_output_plane, filter_size, stride=(1,1), padding=(0,0), mode='cross', with_bias=True, initW=df.init.xavier(), initB=df.init.const(0), border=None):
```

Comment:
Very minor, but maybe we shoud rename `n_input_plane` to `n_input_channels` now that it's not only planes, but also volumes for 3D? And same for output.